src/components/TimePicker/__test__/timePicker.spec.js

```diff
@@ -81,15 +82,15 @@ describe('<TimePicker/>', () => {
             expect(component.find('Modal').prop('isOpen')).toBe(false);
         });
     });
-    it('should call to focusHourInput when the modal is open', () => {
-        const component = mount(<TimePicker label="unit-testing-timePicker" />);
-        component.find('input').simulate('click');
-        component.instance().fieldRef.current.timeSelectRef.current.focusHourInput = jest.fn();
-        setTimeout(() => {
-            expect(
-                component.instance().fieldRef.current.timeSelectRef.current.focusHourInput,
-            ).toHaveBeenCalledTimes(1);
-        }, 0);
+    it('should focus hour input when modal is open', () => {
+        const wrapper = mount(<TimePicker label="unit-testing-timePicker" />);
+        wrapper.find('input').simulate('click');
+        expect(
+            wrapper
+                .find('input')
+                .at(1)
+                .is(':focus'),
```

Comment:
this test is very well, I think we can use a more specific selector to the input like:
```
.find('input[aria-label="hour"]')
```
to be more confident we are selecting the right input since there are four of them